assets/js/components/Loadpoints/SettingsModal.vue

```diff
@@ -67,8 +67,8 @@
 									$t(
 										`main.loadpointSettings.phasesConfigured.phases_${phases}_hint`,
 										{
-											min: minPowerPhases(phases),
-											max: maxPowerPhases(phases),
+											min: fmtW(V() * phases * minCurrent),
+											max: fmtW(V() * phases * maxCurrent),
```

Comment:
Let's keep the calculation in one space. see https://github.com/evcc-io/evcc/pull/22843/files#diff-746a20a7a2917f91ad6cdff159a42db4db4e138dc71eea83a8d22254d74762c4R269

Maybe introduce a `fmtPhasePower(current, phases)` method